src/components/link/Link.tsx

```diff
@@ -0,0 +1,13 @@
+interface LinkProps {
+    href: string;
+    className: string;
+    children: React.ReactNode;
+}
+
+const Link = ({ href, className, children }: LinkProps) => (
+    <a href={href} className={className} target="_blank" rel="noreferrer">
+        {children}
+    </a>
+);
```

Comment:
Try this instead your code, do the same with button component,
Your solution is good to, but if I want in future add a new attrtibute to one of your links, i should extend extend your interface again, so instead you can use a special ComponentType.

https://www.totaltypescript.com/react-component-props-type-helper

```suggestion
const Link = ({ target = "_blank",...restProps }: React.ComponentProps<'a'>) => (
    <a target={target} {...restProps}>
        {children}
    </a>
);
```